JitsiConference.js

```diff
@@ -1450,18 +1450,9 @@ JitsiConference.prototype._acceptJvbIncomingCall = function(
         Statistics.sendEventToAll('session.restart');
     }
 
-    // add info whether call is cross-region
-    let crossRegion = null;
-
-    if (this.options.config && this.options.config.deploymentInfo
-            && typeof this.options.config.deploymentInfo.crossRegion
-            !== 'undefined') {
-        crossRegion = this.options.config.deploymentInfo.crossRegion;
-    }
     Statistics.analytics.sendEvent(
         'session.initiate', {
-            value: now - this.room.connectionTimes['muc.joined'],
-            label: crossRegion
```

Comment:
If cross region is redundant, where else is it getting logged?

Reply:
It is part of config.deploymentInfo, so the client adds it as a permanent property here: https://github.com/jitsi/jitsi-meet/blob/master/react/features/analytics/functions.js#L72